mobify/platform-scaffold
# Integration manager extensibility

This PR contains all the peices for implementing Integration Manager extensibility. Each of these peices have been reviewed before being merged into this branch.

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-78 & https://mobify.atlassian.net/browse/WEBDATA-79


## Changes
- [x] Setup connector extension folder, add registerExtension function to the IM and add a custom commands file to the IM #639 
- [x] Implement a custom command example on merlins #645 
- [x] Implement a command override example on merlins #652 

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- add an item to your cart
- update the item quantity
- the item updates correctly
- proceed to the shipping page
- enter a registered users email address
- the login form is shown with a message stating the user can login
